src/components/tables/TransactionsTable.tsx

```diff
@@ -34,12 +35,17 @@ export type TransactionsTableProps = {
 export default function TransactionsTable({
   account,
 }: TransactionsTableProps): JSX.Element {
+  const [search, setSearch] = React.useState('');
   const [{ pageIndex, pageSize }, setPagination] = React.useState<PaginationState>({
     pageIndex: 0,
     pageSize: 10,
   });
-  const { data: splitsCount } = useSplitsCount(account.guid);
-  const { data: splits } = useSplitsPagination(account.guid, { pageSize, pageIndex });
+  const { data: splitsCount } = useSplitsCount(account.guid, search);
+  const { data: splits } = useSplitsPagination(
+    account.guid,
+    { pageSize, pageIndex },
+    search,
```

Comment:
pass the search term, note the search will be only **within** the interval the user selected